backend/kafka/consumer_vid.py

```
@@ -277,13 +278,20 @@ def process_videos():
         # If it's not a directory, create it
         os.makedirs("./tmp")
         print("Folder /tmp created.")
     else:
         print("Folder /tmp already exists.")
+   
+   
     s3_init()
-    print("Video Consumer is starting...")
+    print("🚦Video Consumer is starting...\n")
+    
+    print("ℹ️ GPU config: ",{"CPU" if gpu_config==None else gpu_config})
```

Ruff (0.15.12)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)